src/pages/webhooks/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Adobe Commerce Webhooks Overview
+description: An architectural overview of Adobe Commerce Webhooks.
+keywords:
+  - Extensibility
+---
+
+# Adobe Commerce Webhooks Overview
+
+Webhooks enable developers to configure synchronous logic to execute calls to external systems when an Adobe Commerce event triggers. Synchronous calls are required when Commerce needs to immediately compute or validate something (order totals, taxes, payments) using a 3rd-party endpoint and write the result back into Adobe Commerce.
+
+This process is illustrated in the following diagram.
+
+![Webhooks architecture](../_images/webhooks.png)
+
+1. Webhook execution always starts with an event occurring in Adobe Commerce. This event can be initiated by any type of user. For example, an event fires when a shopper adds a product to their shopping cart or creates an account. An administrator creates an event when performing tasks such as adding products to the catalog and issuing a refund.
```

Comment:
The last sentence confused me — it took a second to realize that this is intended as another example. The tense changed is a bit confusing, going from events can be initiated, to an event fires, to an administrator creates an event. Our general tone has been that events occur due to the process that users do, rather than people creating events themselves. 

I like having 2 personas in example, maybe we can say something like "This event can occur from many different types of user flows. For example, an event fires when a shopper adds a product to their shopping cart or an administrator creates a quote.